articles/how-to-use-kanban-to-manage-your-next-project/index.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/how-to-use-kanban-to-manage-your-next-project/
+title: How to Use Kanban to Manage Your Next Project
+description: This article will be an introduction to Kanban, which is an agile framework for project management. 
+author: benson-kariuki
+date: 2021-01-15T00:00:00-12:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/how-to-use-kanban-to-manage-your-next-project/hero.jpg
+    alt: Kanban Board Image
+---
+Kanban is an agile framework for product development. It is a visual way of managing tasks and workflow at the individual, team, or organizational level.
+<!--more-->
+
+It helps us to visualize our work even when the work is invisible or intangible. Kanban helps the team visualize the work better, share it with others, and keep everyone on the same page. Kanban improves team collaboration. This article will guide you on how to use Kanban to manage your next project.
+
+### Background of Kanban
+
+Kanban is a Japanese origin word meaning billboard or signboard. Kanban originated in the Toyota factory in 1940. The aim was to manage work while balancing market demands with production capacity. Ever since Kanban has evolved in different generations. Today, Kanban is being used in different industries, including software development. It is applicable in any industry. The software development industry has benefited most from the Kanban framework.
+
+### Introduction to Kanban
+
+Kanban is easy to implement and follow. We start using Kanban by creating a [Kanban board](#kanban-board). The board is filled with [Kanban cards](#kanban-cards), and [work in progress limit(WIP limit)](#work-in-progress-limit-wip-limit) is set. Kanban helps get the work done in an organized way. It does not require teams or organizations to change the way they work. The better part of it is that no new roles are required to start using Kanban. This is one of the Kanban principles. You start with what you currently have. Kanban can also be used at an individual level as a tasks management tool.
```

Comment:
> This is one of the Kanban principles.

I see that you've written about the Kanban principles in your article, below. It would be better if you could add a link that links to the principles that you've listed below.